client/messaging/message-records.ts

```diff
@@ -1,75 +1,37 @@
 import { Record } from 'immutable'
+import { ChatMessage } from '../chat/chat-message-records'
+import { LobbyMessage } from '../lobbies/lobby-message-records'
 
-export enum ChatMessageType {
+/**
+ * NOTE(2Pac): A common message type that's used in all messaging-related services (e.g. chat,
```

Comment:
Not sure why this enum documentation is in the form of a `NOTE(...):`, you can just document it normally :)